src/components/ContentNav/index.tsx

```diff
@@ -42,32 +43,35 @@ export class ContentNav extends Component<ContentNavProps, ContentNavState> {
       const isActive = item.name === this.props.activeElement;
       let newItem = { ...item, active: isActive };
       let subItemIsActive = false;
-      if (item.subItems) {
-        newItem.subItems = item.subItems.map(subItem => {
-          subItemIsActive = `${item.name}-${subItem.name}` === this.props.activeElement;
+      newItem.subItems = item.subItems.map(subItem => {
+        subItemIsActive = `${item.name}-${subItem.name}` === this.props.activeElement;
 
-          return { ...subItem, active: subItemIsActive };
-        });
+        return { ...subItem, active: subItemIsActive };
+      });
 
-        newItem = { ...newItem, expanded: subItemIsActive };
-      }
+      newItem = { ...newItem, expanded: subItemIsActive };
 
       return newItem;
     });
   }
 
   subItemClicked = (event: React.MouseEvent<HTMLLIElement>, name: string) => {
     event.stopPropagation();
+    Scroll.scroller.scrollTo(name, {
```

Comment:
This function call can be extracted into it's own scope, we have some repeated code here and if we defined a function in this component called `scrollTo` we could pass in the name of the section as an argument and run  `scroller.scrollTo`.

Reply:
This has been updated.